src/main/kotlin/no/nav/dagpenger/grunnbelop/Grunnbelop.kt

```diff
@@ -8,6 +8,7 @@ import java.time.YearMonth
 
 enum class Grunnbeløp(val verdi: BigDecimal, val iverksattFom: LocalDate) {
     GjusteringsTest(verdi = 124028.toBigDecimal(), iverksattFom = LocalDate.now().plusYears(10)),
+    FastsattI2024(verdi = 124028.toBigDecimal(), iverksattFom = LocalDate.of(2024, Month.JUNE, 3)),
```

Comment:
Blir ikke `iverksattFom = LocalDate.of(2024, Month.MAY, 31))` ? Det er vel natt til søndag prodsettinga skal være @androa ?

Reply:
Jo, det har du nok helt rett i

Reply:
Eller, G-reguleringa gjøres vel natt til lørdag i år.

Reply:
Ja, jo. jeg så feil da. 2. juni blir det vel. 31 mai er jo fredag.

Reply:
Var jo litt hummer og kanari på om vi har satt lørdag eller søndag før ser jeg 😓

Reply:
Det er vel ikke så veldig viktig, så lenge den er gyldig ved G-justeringstidspunktet.

Reply:
Som blir 1. juni dette tilfellet 😓

Reply:
Ja, tror det er ryddigst å sette søndag, da kan en deploye og gjøre ferdig alt

Reply:
Det må vel bli lørdag, slik at denne G-en finnes når Arena kaller oss for å beregne nytt grunnlag? Denne overstyrer vel uansett det vi putter i gyldighetsperioder.